demisto_sdk/commands/doc_reviewer/rn_checker.py

```diff
@@ -48,7 +47,11 @@ class ReleaseNotesChecker:
 
         # full line
         'Documentation and metadata improvements.',
-        'Maintenance and stability enhancements.',
+    }
+
+    BANNED_TEMPLATES = {
```

Comment:
Discussed this with @bakatzir - even though you are right and you can bypass this validation easily, it will help people to become more aware of the goal to avoid using such phrases.